src/Features/CSharp/Portable/CodeRefactorings/SyncNamespace/CSharpChangeNamespaceService.cs

```diff
@@ -95,6 +96,8 @@ protected override SyntaxList<MemberDeclarationSyntax> GetMemberDeclarationsInCo
             throw ExceptionUtilities.Unreachable;
         }
 
+#nullable enable
```

Comment:
can we just turn this on/off at the file level?  i'd really like to avoid a world where we just have all these #nullable scattered all over every file.

Reply:
Since this is mop-up work, yeah, let's just annotate the entire file.

Reply:
Done